### PR TITLE
Capture: Stop a capture when an input recording is finished

### PR DESCRIPTION
**Pull request by sonicfind**

### CODE REVIEW

**Review by RedDevilus — APPROVED** [review]

**Comment by RedDevilus** [comment]
image

Guess it doesn't like multiple PCSX2 instances running, no idea how i triggered that when i wanted to close it.

**Review by GovanifY — COMMENTED** [review]
LGTM, just have a small question

File: pcsx2/Recording/InputRecordingControls.cpp [hunk 1]
```diff
@@ -46,12 +46,12 @@ void InputRecordingControls::CheckPauseStatus()
 			g_InputRecording.SetToReplayMode();
 			switchToReplay = false;
 		}
-
-		if (!pauseEmulation &&
-			g_InputRecording.IsReplaying() &&
-			g_InputRecording.GetFrameCounter() >= g_InputRecording.GetInputRecordingData().GetTotalFrames())
+		
+		if (IsFinishedReplaying() || g_InputRecording.GetFrameCounter() == INT_MAX)
```
[the remark this answers is not in the document]
Comment (GovanifY) [reply pcsx2/Recording/InputRecordingControls.cpp:50]: Thanks for confirming!

**Review by GovanifY — APPROVED** [review]

**Comment by RedDevilus** [comment]
Could you rebase please?

**Comment by sonicfind** [comment]
Ditto

**Review by xTVaser — APPROVED** [review]
Still works as expected.  The libchdr submodule change seems out of place though

**Pull request merged**